### ISSUE TITLE
Ensemble of two models

### ISSUE THREAD

**Issue by uzielroy** [LiYingwei/ShapeTextureDebiasedTraining#4]
Can you please provide me detalis regarding the ensemble of two models?
1. During training: did you just compute one loss for each model?
2. During testing: how did you get the final logits?
Thanks!

**Comment by LiYingwei** [comment]
I assume you are asking about the model ensemble mentioned in [our response to AnonReviewer2](https://openreview.net/forum?id=Db4yerZTYkz&noteId=KQ32n5w-XU-) Question3. Correct me if my understanding is incorrect.

For the training stage, your understanding is correct: two models are training separately.

For the testing stage, the final logits are computed by averaging the two logits vectors that from two models.

Feel free to let me know if you have further questions.
